### PR TITLE
Move init script logic into build

### PR DESCRIPTION
**Pull request by blindpirate**
This PR removes unnecessary init script and moves it into build.

### CODE REVIEW

**Comment by blindpirate** [comment]
@bot-gradle test this

File: buildSrc/subprojects/profiling/src/main/kotlin/gradlebuild/buildscan.gradle.kts [hunk 1]
```diff
@@ -27,10 +27,10 @@ import org.jsoup.parser.Parser
 import java.net.URLEncoder
 import java.util.concurrent.atomic.AtomicBoolean
 
-val serverUrl = "https://e.grdev.net"
+val serverUrl = "https://ge.gradle.org"
 val gitCommitName = "Git Commit ID"
 val ciBuildTypeName = "CI Build Type"
-val watchFileSystemName = "watchFileSystem"
```
Comment (jjohannes) [inline buildSrc/subprojects/profiling/src/main/kotlin/gradlebuild/buildscan.gradle.kts:33]: I probably added this on merge conflict. Thanks for cleaning up!

File: buildSrc/subprojects/profiling/src/main/kotlin/gradlebuild/buildscan.gradle.kts [hunk 2]
```diff
@@ -27,10 +27,10 @@ import org.jsoup.parser.Parser
 import java.net.URLEncoder
 import java.util.concurrent.atomic.AtomicBoolean
 
-val serverUrl = "https://e.grdev.net"
+val serverUrl = "https://ge.gradle.org"
```
Comment (jjohannes) [inline buildSrc/subprojects/profiling/src/main/kotlin/gradlebuild/buildscan.gradle.kts:30]: Did I break this on merge conflict? Sorry.. :(

**Comment by blindpirate** [comment]
Thanks @jjohannes !

**Pull request merged**